frontend/packages/topology/src/layouts/ColaLayout.ts

```
@@ -98,60 +113,61 @@ class ColaLink implements webcola.Link<ColaNode | number> {
   }
 }
 
-class ColaGroup implements webcola.Group {
-  public bounds?: webcola.Rectangle;
-
-  public leaves?: ColaNode[];
-
-  public groups?: ColaGroup[];
-
-  public padding: 200;
-
-  constructor(children: NodeEntity[] | undefined, allNodes: ColaNode[]) {
-    this.leaves = _.reduce(
-      children,
-      (nodes: ColaNode[], nodeEntity: NodeEntity) => {
-        const nextChild: ColaNode | undefined = allNodes.find(
-          (nextNode: ColaNode) => nextNode.id === nodeEntity.getId(),
-        );
-        if (nextChild) {
-          nodes.push(nextChild);
-        }
-        return nodes;
-      },
-      [],
-    );
-  }
-}
+const getNodeIndex = (nodes: ColaNode[], id: string): number => {
+  const node = _.find(nodes, { id });
+  return node ? node.nodeIndex : 0;
```

is index 0 correct? If not found, shouldn't it be `-1` ?
Or will it _always_ be found? in which case it should throw an error if not.